### PR TITLE
Cleanup for Labs

### LINKED ISSUE #3131: Top-bar symbols not properly sized
**Describe the bug**

The top bar symbols are not of adequate size.
Precisely, Gitcoin symbol and Codefund symbol are being cropped.

image

**To Reproduce**

Just open gitcoin.co on any mobile device.

**Expected behavior**

image

**Screenshots**
Provided above

**Desktop (please complete the following information):**

None.

**Smartphone (please complete the following information):**

All devices.

**Additional Context**

I would love to work on this issue.
I have attached expected behavior by fixing this on my branch for #3048 that's why we see Labs there too.

### PR DESCRIPTION
**Pull request by aashutoshrathi**
##### Description

- Add Upcoming (boolean) to Labs Model for Roadmaps
- Added one more banner with Mr. Austin to cover nakedness.
- Waiting for new Topbar SVG for labs.

##### Checklist

- [x] Read and conforms to the [contributor guidelines](https://docs.gitcoin.co/mk_contributors/)
- [x] Read and conforms to the [style guidelines](https://docs.gitcoin.co/mk_styleguide/)
- [x] Linter status: 100% pass
- [x] Added relevant tests
- [x] Tests pass and code coverage has not decreased
- [x] Changes don't break existing behavior
- [x] Commit message follows [commit guidelines](https://docs.gitcoin.co/mk_contributors/#step-4-commit)
- [x] [Code documentation](https://docs.gitcoin.co/mk_contributors/#docstrings)
- [x] Completed: Code review by core team and any requested changes

##### Refers/Fixes
Fixes #3131 and #3184 and changes requested in #3048 by @owocki 

###### Contributor

- [x] Read and followed the [Contributor Guidelines](https://docs.gitcoin.co/mk_contributors/)
- [x] Tested all changes **locally**
- [x] Verified existing functionality
- [ ] Ran `make test` and everything passed!

### CODE REVIEW

**Comment by aashutoshrathi** [comment]
Code related work is in this commit: https://github.com/gitcoinco/web/pull/3227/commits/4e6ac8d96d4d98871dd1c342725f7ac7d67b861d

SVG fixes can be seen in Mobile View and Rest in both.

Mobile | PC
--- | --- 
fix-mob | fix-web

File: app/dashboard/templates/labs.html [hunk 1]
```diff
@@ -115,6 +116,12 @@ <h3 class="sub-title dark-text">Austin Griffith</h3>
                 <i class="fab fa-medium fa-lg"></i>
               </a>
             </div>
+
+            <div class="mx-lg-auto col-lg-6 col-xl-3 mt-3 text-center pl-xl-0 ml-xl-0">
+              <h1 class="dark-text">You?</h1>
+              <p class="description">Want to join the Labs team? Start contributing on <a href="https://github.com/gitcoinco/" class="font-weight-bold magenta-link">GitHub</a>.
```
Comment (mbeacom) [inline app/dashboard/templates/labs.html]: All new copy needs wrapped in `{% trans "" %}`

File: app/dashboard/views.py [hunk 2]
```diff
@@ -1413,7 +1413,8 @@ def toolbox(request):
 
 
 def labs(request):
-    labs = LabsResearch.objects.all()
+    labs = LabsResearch.objects.filter(upcoming=False)
+    roadmap = LabsResearch.objects.all()
```
Comment (mbeacom) [inline app/dashboard/views.py]: Why are we querying for both of these?
Can we eliminate one or handle the `upcoming` check in template?

File: app/assets/v2/images/vision/triangle.svg [hunk 3]
```diff
@@ -1 +1 @@
....495-1.63 3.768v.005c-.029 1.57.672 3.187 1.922 4.438l.168.17a5.645 5.645 0 0 0-1.44 1.052c-1.032 1.031-1.613 2.374-1.637 3.78-.027 1.572.681 3.196 1.94 4.454l1.96 1.96a5.772 5.772 0 0 0-1.462 1.07 5.436 5.436 0 0 0-1.619 3.76c-.028 1.572.673 3.19 1.922 4.44l14.496 14.494-6.236 6.234-28.437-28.439a17.935 17.935 0 0 1-5.288-12.766c0-9.956 8.1-18.054 18.054-18.054 4.823 0 9.357 1.877 12.767 5.287l2.63 2.63a.406.406 0 0 0 .555 0l2.628-2.63a17.94 17.94 0 0 1 12.767-5.287c9.955 0 18.055 8.098 18.055 18.054a18 18 0 0 1-.552 4.35l-6.354-6.354zm10.168 10.167l-.798-.797a21.55 21.55 0 0 0 1.288-7.366c0-12.025-9.782-21.807-21.807-21.807a21.659 21.659 0 0 0-15.42 6.387l-.253.253-.254-.253a21.655 21.655 0 0 0-15.42-6.387c-12.024 0-21.806 9.782-21.806 21.807a21.667 21.667 0 0 0 6.386 15.42l24.318 24.32c.011.007.02.017.031.026l6.466 6.466a.395.395 0 0 0 .556 0l8.635-8.639c4.49 4.2 9.813 6.513 15.136 6.513.112 0 .226 0 .337-.003 5.131-.092 10.027-2.213 13.79-5.973 3.748-3.75 5.863-8.642 5.956-13.77.105-5.736-2.433-11.488-7.14-16.197z" fill="#180041"/><path fill="#00ED92" d="M163.265 311.441l-5.583-9.875 10-22.375 17-.25-3.625 20.625zM485.765 279.191h22.5l6.834 20.833-7.334 13.167z"/></g></g></svg>
```
Comment (mbeacom) [inline app/assets/v2/images/vision/triangle.svg]: Can you please cherry pick your changes and omit all of the changes to these `.svg` assets?

File: app/dashboard/templates/labs.html (hunk 1 above)
Comment (aashutoshrathi) [reply app/dashboard/templates/labs.html]: Sure

File: app/assets/v2/images/vision/triangle.svg (hunk 3 above)
Comment (aashutoshrathi) [reply app/assets/v2/images/vision/triangle.svg]: Sure

File: app/dashboard/views.py (hunk 2 above)
Comment (aashutoshrathi) [reply app/dashboard/views.py]: Sure

**Comment by aashutoshrathi** [comment]
@mbeacom Done.

**Comment by aashutoshrathi** [comment]
@mbeacom Done

**Comment by aashutoshrathi** [comment]
@thelostone-mc https://github.com/gitcoinco/web/pull/3227#issuecomment-447675396

**Comment by thelostone-mc** [comment]
@aashutoshrathi could you reduce the size of You a bit and give it a lil breathing space ?

**Comment by aashutoshrathi** [comment]
Sure @thelostone-mc just a second.

**Comment by aashutoshrathi** [comment]
ping @thelostone-mc

**Comment by thelostone-mc** [comment]
@mbeacom you good with this ?

**Comment by aashutoshrathi** [comment]
@thelostone-mc I updated branch using CLI, that dismissed your review, please check

**Pull request merged**